lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -55,7 +33,7 @@ public function __construct() {
 	public function register_routes() {
```

Comment:
Still overriding `register_routes` for font faces, as we don't want the `WP_REST_Server::EDITABLE` route that's created by default for updating existing font faces (which could get quite complex with file uploads)